Formula/aptos.rb

```diff
@@ -0,0 +1,34 @@
+class Aptos < Formula
+  desc "Layer 1 blockchain built to support fair access to decentralized assets for all"
+  homepage "https://aptoslabs.com/"
+  url "https://github.com/aptos-labs/aptos-core/archive/refs/tags/aptos-cli-v1.0.3.tar.gz"
+  sha256 "670bb6cb841cb8a65294878af9a4f03d4cba2a598ab4550061fed3a4b1fe4e98"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    regex(/^aptos-cli[._-]v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "rustup-init" => :build
+  uses_from_macos "llvm" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "openssl@1.1"
```

Comment:
In the [documentation for the openssl crate](https://docs.rs/openssl/latest/openssl/#automatic) we use it seems like they specify `openssl 1.1` specifically:

```
brew install openssl@1.1
```

but I can give it a try with openssl@3 and see what happens

Reply:
@chenrui333 looks like it passed

Reply:
I think there is docs opportunity there :)

Reply:
this would be a better reference, they look for both.
https://github.com/sfackler/rust-openssl/blob/8621407a3268071e30312dfe40098b574554ea1f/openssl-sys/build/find_normal.rs#L36

Reply:
Nice!